packages/hint-doctype/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "hint to validate if the doctype is correct",
+  "devDependencies": {
+    "@hint/utils-tests-helpers": "^1.0.0",
+    "@types/debug": "0.0.30",
+    "@types/node": "8.0.14",
```

Comment:
Now that we are in strict mode we can update this version. Can you make it match the one we have in the rest of the project? I don't think it's the latest one though.

Reply:
@CKGrafico `@types/node` is still an old version.